### ISSUE TITLE
New canvas

### ISSUE THREAD

**Issue by arvind-prasad68** [biolab/orange3#3099]
##### Orange version
3.13

##### Expected behavior
Whenever a widget is loaded on a canvas, the features should be set to a default value

##### Actual behavior
The feature selection of the last saved canvas is loaded on selection of a new widget.

**Comment by ajdapretnar** [comment]
This is a feature. If you wish to start with default Orange, select Options - Reset widget settings.

**Comment by kernc** [comment]
@arvind-prasad68 Why does this inconvenience you?

**Comment by arvind-prasad68** [comment]
kernc, in most of the simulation softwares if I drag a new instance of an object, the features come with default settings. This is convenient, as you know the default settings, and you are required to change the setting as per your requirements. If the last used settings are loaded, one might unconsciously assume that default settings are being used, and this may lead to errors. For example, when I was using linear regression widget, I selected the ridge regularization and set the alpha. Later on for a fresh problem, when I loaded a new instance of linear regression widget, the regularization was set to the last used regularization. Each problem, you will appreciates require a new kind of regularization with different values of paramerters like Alpha.
